backend/src/backend/main.py

```diff
@@ -51,4 +53,28 @@ async def test(timestamp: int):
     return {"requested_on": requested_on, "received_on": received_on}
 
 
+@api.post("/books/add/", status_code=201)
+async def books_add(book: SchemaBook):
+    await Book.objects.create(
```

Comment:
You may want to wrap this all into a transaction so that we don't end-up with partial data in case something goes wrong later-on as this is a multi-stage, multi-table insertion